src/components/formWrapper/FormWrapper.vue

```diff
@@ -70,15 +70,15 @@ export default {
   },
   mounted() {
     this.currentTab = formWrapperTabs[0]
-    this.globalStore.tokenTypeSelected = this.currentTab.id
+    this.globalStore.actualTokenType = this.currentTab.tokenType
```

Comment:
I'd change 'actual' for 'current'

Reply:
done at https://github.com/rsksmart/tokenbridge-ui/pull/52/commits/71e1d66d07a4e1ec59e8323e2504f1d9caa3465d